nautobot/core/settings.py

```diff
@@ -130,7 +130,9 @@
 
 # CRYPTOGRAPHY_BACKEND = cryptography.hazmat.backends.default_backend()
 # CRYPTOGRAPHY_DIGEST = cryptography.hazmat.primitives.hashes.SHA256
-CRYPTOGRAPHY_KEY = None  # Defaults to SECRET_KEY if unset
+# CRYPTOGRAPHY_KEY defaults to SECRET_KEY if unset;
+# this is undesirable, see https://github.com/nautobot/nautobot/issues/502
+CRYPTOGRAPHY_KEY = os.getenv("CRYPTOGRAPHY_KEY")
```

Comment:
```suggestion
CRYPTOGRAPHY_KEY = os.getenv("NAUTOBOT_CRYPTOGRAPHY_KEY")
```

Reply:
Also, just wanted to be explicit that we want this to fallback to `None`.

Reply:
Good catch on the variable name! Though oddly we use `SECRET_KEY = os.environ.get("SECRET_KEY")` in this file, which seems inconsistent at least.

It falls back to None, which `django-cryptography` interprets to mean "use the `SECRET_KEY` instead", which will be caught and rejected by the new check I added.

Reply:
> Though oddly we use SECRET_KEY = os.environ.get("SECRET_KEY") in this file, which seems inconsistent at least.

That definitely predates us revising everything to be prefixed with `NAUTOBOT_`.